708 add slider to cart

dstopp opened the pull request:
Add slider to the cart view + responsiveness.
-Slider behaviour on cart should mimic behaviour on browse tab
-Slider should use max count from number of items in the cart
-Slider should gray out when there are not enough items on the page to slider

dstopp committed:
-add slider to cart page
-change id to classes for support of new slider
under construction

dstopp committed:
initial stab at slider responsiveness

dstopp committed:
some additional changes to the css for the cart slider

dstopp committed:
Add responsiveness to the cart slider <thanks, Dave!>

rfrenchseti commented on opus/application/static_media/js/opus.js:
```diff
@@ -196,7 +196,7 @@ var opus = {
 
         // Start the observation number slider spinner - no point in doing a flash here
-        $("#browse .op-observation-number").html(opus.spinner);
+        $(".op-observation-number").html(opus.spinner);
```
Are you sure you want to do this for both sliders? Just because the search has changed doesn't mean we need a spinner on the cart. They are unrelated.

dstopp replied:
good catch, updating to just set the spinner on the #browse tab.

rfrenchseti reviewed (changes requested):
- Browse slider is very broken. It's short, fat, has no space above it, and no horizontal line.
- On the cart tab, there's no vertical space between the slider and the sort pills when the slider is on the second row.

dstopp committed:
-fixed the slider on the browse tab
-adjusted the spacing between the slider handle and the sort pills on the cart page

rfrenchseti reviewed (changes requested):
Bug #1:
- Put enough observations in the cart that you can enlarge your browser and not quite fill the page
- Note that the cart slider is greyed out
- Gradually shrink the browser. At some point the slider will no longer be greyed out, and will say "1" in a narrow slider handle (narrower than the max number of obs in the cart). The slider can't be moved.
- Narrow the slider further. At some point the handle will widen to the proper width to hold the total number of obs in the cart, and the slider will be moveable.
- This implies to me that the logic to disable the slider is not working properly.

Bug #2:
- When the slider is on the second row, there is no space between the slider handle and the sort pills. There should be the same space as on the browse tab.

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 708_add_slider_to_cart

rfrenchseti reviewed (changes requested):
There is still no space between the slider handle and the sort order pills on the cart page. It doesn't look like there has been a new push in the past week.

Also there is a responsiveness problem on the cart page. At width 1190px - 1210px there is an extra blank line added between the slider and the sort pills.

dstopp committed:
-fixed  space between the slider handle and the sort order pills on the cart page

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 708_add_slider_to_cart

dstopp committed:
Merge branch 'test_branch_with_different_slider_behavior_based_on_slider_issue_807_and_infiniteScroll_issue_761' of https://github.com/juzen2003/pds-opus into 708_add_slider_to_cart

# Conflicts:
#	opus/application/static_media/js/browse.js

dstopp committed:
Merge branch 'test_branch_with_different_slider_behavior_based_on_slider_issue_807_and_infiniteScroll_issue_761' of https://github.com/juzen2003/pds-opus into 708_add_slider_to_cart

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 708_add_slider_to_cart

dstopp committed:
-make the label View Table and View Gallery on the navbar the same width
-update/standardize the selectors

dstopp committed:
-add a bit of space under the slider when the screen narrows.

dstopp committed:
- fix the left pad on #browse .gallery

dstopp committed:
added mutiations for cart; they were only set up for the first item in the array which happened to be browse.
this fixes the issue w/the slider handle max value being incorrect because the mutiation occurs, which allows the table size to be calculated correctly.

dstopp committed:
-remove unnecessary comments
-don't enable spinner for cart on change of search parameters.

dstopp committed:
fixed browse tab css spacing issue

dstopp committed:
fixed blank line

dstopp committed:
-fix for issue #821; rename download options, move to second line, add responsiveness
Note that the download links are now blue; that is intentional, please comment if preference is to use the white nav color instead.

dstopp committed:
renamed class to op- for sort order container classes.

dstopp committed:
-change the order of the lines in the cart nav so that the download options are last;
-change the color of the download options so they stand out a little bit better
-update the responsiveness

dstopp committed:
- change the width of the slider at break 1300

dstopp committed:
fixed download option issue at break 930

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 708_add_slider_to_cart

# Conflicts:
#	opus/application/static_media/js/browse.js

note - review line 718 in browse.js

dstopp committed:
-fixed the empty panel between 901 & 929
-fixed the bug that left behind an empty panel when the width of the cart increased enough to not need the panel
-removed code at line 722, may be merge issue.
-added addition break at 1425 to adjust slider container size; probably should be done in JS instead of css for smoother size adjustment

dstopp committed:
-changed the break in the cart from 930 to 940 to account for font size diff in linux

dstopp committed:
-set the size of the cart nav container such that the slider will always be at least a certain distance apart from the empty cart button

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 708_add_slider_to_cart

# Conflicts:
#	opus/application/apps/ui/templates/ui/base.html
#	opus/application/static_media/js/cart.js
